tests/osaka/eip7692_eof_v1/eip4750_functions/test_callf_execution.py

```diff
@@ -22,6 +22,85 @@
 pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
 
 
+def test_callf_factorial(eof_state_test: EOFStateTestFiller):
+    """Test factorial implementation with recursive CALLF instructions"""
+    eof_state_test(
+        data=Container(
+            sections=[
+                Section.Code(
+                    Op.CALLDATALOAD(0) + Op.SSTORE(0, Op.CALLF[1]) + Op.STOP,
+                    max_stack_height=2,
+                ),
+                Section.Code(
+                    Op.PUSH1[1]
+                    + Op.DUP2
+                    + Op.GT
+                    + Op.RJUMPI[4]
+                    + Op.POP
+                    + Op.PUSH1[1]
+                    + Op.RETF
+                    + Op.PUSH1[1]
+                    + Op.DUP2
+                    + Op.SUB
+                    + Op.CALLF[1]
+                    + Op.DUP2
+                    + Op.MUL
+                    + Op.SWAP1
+                    + Op.POP
+                    + Op.RETF,
+                    code_inputs=1,
+                    code_outputs=1,
+                    max_stack_height=3,
+                ),
+            ]
+        ),
+        tx_data=(57).to_bytes(32, "big"),
+        container_post=Account(
+            storage={0: 0x59996C6EF58409A71B05BE0BADA2445EB7C017D09442E7D158E0000000000000}
+        ),
+    )
+
+
+def test_callf_fibonacci(eof_state_test: EOFStateTestFiller):
+    """Test fibonacci sequence implementation with recursive CALLF instructions"""
+    eof_state_test(
+        data=Container(
+            sections=[
+                Section.Code(
+                    Op.CALLDATALOAD(0) + Op.SSTORE(0, Op.CALLF[1]) + Op.STOP,
+                    max_stack_height=2,
+                ),
+                Section.Code(
+                    Op.PUSH1[2]
+                    + Op.DUP2
+                    + Op.GT
+                    + Op.RJUMPI[4]
+                    + Op.POP
+                    + Op.PUSH1[1]
+                    + Op.RETF
+                    + Op.PUSH1[2]
+                    + Op.DUP2
+                    + Op.SUB
+                    + Op.CALLF[1]
+                    + Op.PUSH1[1]
+                    + Op.DUP3
+                    + Op.SUB
+                    + Op.CALLF[1]
+                    + Op.ADD
+                    + Op.SWAP1
+                    + Op.POP
+                    + Op.RETF,
+                    code_inputs=1,
+                    code_outputs=1,
+                    max_stack_height=4,
+                ),
+            ]
+        ),
+        tx_data=(13).to_bytes(32, "big"),
```

Comment:
same request, parametrize a few interesting values (0, 1, 6, 7).  But the biggest would be at 15m/30m gas since this is the fully expanded version of the calculation?